.ai/PLAN.md

```
@@ -0,0 +1,204 @@
+- Nextor should allow to format floppy disks.
+- The single sided/double sided disks shenanigans should be supported.
+- MSX-DOS drivers included a "stop motor" feature that should be made available too.
+- The "ghost drive" mechanism, which allows to transfer files between two disks when one single drive is available, should be present too.
+
+Details on each of these points follow.
+
+## Identifying a floppy disk drive device
+
+The "Get device parameters" device query currently returns:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)